data/README.md

```diff
@@ -0,0 +1,68 @@
+
+This directory contains files used for pre-loading our production server
+database with *Pseudomonas aeruginosa*-related samples and annotations. All of
+these files are loaded into the system using the management commands listed in
+[the load_default_pseudomonas_data.sh script](https://github.com/greenelab/adage-server/blob/master/load_default_pseudomonas_data.sh).
+These commands take the location and filename of the desired data files as
+an argument.
+
+If using [fabric](http://www.fabfile.org/), these can also be loaded using the
+`import_data_and_index()` function in
+[fabfile/adage_server.py](https://github.com/greenelab/adage-server/blob/master/fabfile/adage_server.py).
+The
+[adage/adage/config.py.template file](https://github.com/greenelab/adage-server/blob/master/adage/adage/config.py.template)
+specifies which files are loaded by this `import_data_and_index()` function.
+
+All of the files below are tab-delimited text files (the decompressed form
+of the gzipped file `gene_gene_network_cutoff_0.2.txt.gz` is a tab-delimited
+text file as well). If you wish to load your own files into your adage-server
+instance, they should be in the same format as the files below.
+
+## File Listing and Description
+
+### **PseudomonasAnnotation.tsv**
+  Lists samples retrieved from ArrayExpress along with manually-curated
+  annotations for those samples. Each row is a sample, and the columns
+  contain information about that sample.
+
+### **all-pseudomonas-gene-normalized.pcl**
+  Contains normalized expression levels for every gene in each sample in
+  the compendium. In this file, genes are rows, and samples are columns.
+  This is the input expression data for building an ADAGE model.
+
+### **sample_signature_activity.txt**
+  Contains signature (node) activity levels (as absolute values) for each
+  sample in the compendium derived from the eADAGE machine learning model.
+  In this file, samples are rows, and signatures are columns. This file is
+  based on the
+  `all-pseudomonas-gene-normalized_HWActivity_perGene_with_net300_100models_1_100_k=300_seed=123_ClusterByweighted_avgweight_network_ADAGE.txt`
+  file, which is now in the `data/old/` folder.
+
+### **gene_gene_network_cutoff_0.2.txt.gz**
+  Contains gene-gene network information derived from the eADAGE machine
+  learning model, where each row contains the edge weight between two genes
+  in this model and whether this is a positive or negative weight. Because
+  of its size, this file has been compressed using
+  [gzip](http://www.gzip.org/). However, its decompressed content is a
+  tab-delimited text file as well. 
+
+  The first two columns are the genes for the given edge weight. The order of
+  these first two columns ("from"/"to") does not matter, as the weight of the
+  edge between the two genes is the same. The third column is the edge's
+  weight, and the fourth column indicates whether the weight is positive or
+  negative.
+
+### **signature_gene_network.txt**
```

Comment:
I think this file was generated using the script `write_HWG.R` here https://bitbucket.org/greenelab/eadage/src/d8a98586e0f448a5b7d497f5516cc4e241635361/node_interpretation/write_HWG.R?at=default&fileviewer=file-view-default. However, the output of `write_HWG.R` has one signature per column. Here it's one signature per row. Not sure what happened in the middle that transformed it. Yes, it does look like GMT file format after transformation. However, the GMT file has a description column between gene set name and genes. I don't think we support a description column but I could be wrong.

Reply:
@tj8901nm: When I started to work on the signature-gene network data uploading sometime last year, I was given a bunch of data files, each of which corresponds to the list of heavy genes related to only one signature (node). In order to speed up the uploading process, I merged these files into one. Then changed the signature/node name from "Node###" to "Sig###".

Reply:
Hmm, OK. I think it would be good to keep the input files to ADAGE server consistent with the output files from the eADAGE repo. In this way, users can build an ADAGE model using the eADAGE repo and then start an ADAGE server from those files. I am fine with how this is set up for now but we should consider this in the future.